docassemble/clcsa-01-service-directory/data/static/CLCSA_CSS_Website.css

```diff
@@ -200,3 +200,12 @@ body.da-pad-for-footer {
     height: 24px !important;
   }
 }
+
+.navbar-toggler {
+  background-image: none;
+  border-width: 0px;
+}
+
+.navbar-toggler-icon {
+  background-image: none;
+}
```

Comment:
This doesn't remove the menu entirely,
It should be:
```css
.navbar-toggler {
  display: none;
}
```

But I suggest you use the JS code in #20 so remove this CSS.
```suggestion

```